### PR TITLE
Chore/case studies

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

New case studies section

case-studies

Screenshot 2023-03-21 at 12 00 56

### CODE REVIEW

File: apps/www/_case-studies/epsilon3.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+author_title: Supabase
+author_url: https://github.com/kiwicopple
+author_image_url: https://avatars2.githubusercontent.com/u/10214025?s=400&u=c6775be2ae667e2acae3ccd347fed62bb3f5b3e7&v=4
+authorURL: https://github.com/kiwicopple
+logo: /images/case-studies/logos/epsilon3.png
+tags:
+  - supabase
+date: '28-10-2020'
+company_url: 'https://epsilon3.io'
+stats:
```
Comment (RamiroND) [inline apps/www/_case-studies/epsilon3.mdx:14]: Since we don't have any stat, should we remove this component from the old case studies?

**Review by RamiroND — CHANGES_REQUESTED** [review]

File: apps/www/_case-studies/epsilon3.mdx (hunk 1 above)
Comment (MildTomato) [reply apps/www/_case-studies/epsilon3.mdx:14]: yea sure

File: apps/www/_case-studies/epsilon3.mdx [hunk 2]
```diff
@@ -0,0 +1,94 @@
+---
+title: Epsilon3 digitize paper-based procedures in the space industry using telemetry data, simplifying testing and operations.
+description: Epsilon3 uses Supabase to help teams execute secure and reliable operations in an industry where project spend runs into the billions.
+author: rory_wilding
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_case-studies/epsilon3.mdx:4]: i'll hide them

File: apps/www/_case-studies/xendit.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### What they built
+
+Xendit parses international sanctions lists from the UN and the Indonesian government and loads them into Supabase. Since Supabase provides a full Postgres server, they can then use the [Trigram](https://www.postgresql.org/docs/current/pgtrgm.html) extension to perform full-text search on the lists, with a relevance score on every search.
+
+Supabase was perfect for their use case, as they needed something built fast. The full solution was built and in production in less than one week.
+
+> The full solution was built and in production in less than one week.
+
+Xendit created a database function for searching, which they are able to call directly using their Python clients. They have plans to iterate on the current implementation using more advanced techniques, like machine learning, but for now the Supabase system has been in Production for 9 months without a problem.
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_case-studies/xendit.mdx:40]: Going to ship anyway. pages are hidden for now

File: apps/www/_case-studies/epsilon3.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_case-studies/epsilon3.mdx:4]: done

File: apps/www/_case-studies/epsilon3.mdx (hunk 1 above)
Comment (MildTomato) [reply apps/www/_case-studies/epsilon3.mdx:14]: hidden for all for now

**Review by RamiroND — APPROVED** [review]

**Pull request closed without merging**